react/README.md

```diff
@@ -70,6 +70,14 @@ If you want to run a copy of Gatsby locally:
 2. Run `npm start`.
 3. Open your browser to `http://localhost:8000/`.
 
+If you want to run a copy of Gatsby and Storybook locally:
+
+1. Run `npm run installall`.
+2. Run `npm start` to start Gatsby then open your browser to `http://localhost:8000/`.
+3. Run `(cd react/ ; npm run storybook)` to start Storybook for React.
```

Comment:
for clarity, I would separate these into different back ticks instead of combining them. Especially since you don't really know what directory they're on.

Something like:
> Go into the `react/` folder (`cd react`) and then run the command to start Storybook (`npm run storybook`)

This change would be for all steps and every readme (HTML, React, Angular)

Reply:
If you type this whole thing `(cd react/ ; npm run storybook)` in the terminal from the root directory it will run the command in the other directory without needing to move to that directory.

Reply:
Even though it feels obvious, it's possible that they're not in the root directory. As I beginner, I would appreciate less shortcuts so I know exactly what I'm doing and where I'm doing it. 

If you feel strongly about this, I'm at a 6 about it.

Reply:
I added a line to clarify that you should be in the root directory.